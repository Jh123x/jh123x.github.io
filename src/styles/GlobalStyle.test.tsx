import * as React from "react";
import { describe, expect, test } from "@jest/globals";
import GlobalStyle from "./GlobalStyle";

describe("GlobalStyle", () => {
  test("should render correctly", () => {
    const globalStyle = <GlobalStyle />;
    expect(globalStyle).toMatchSnapshot();
  });
});
