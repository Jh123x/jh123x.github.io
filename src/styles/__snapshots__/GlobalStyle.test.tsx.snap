// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalStyle should render correctly 1`] = `<Memo(a) />`;
